src/site/data/Assignment2/test/DfsMazeSolverTest.java

```diff
@@ -6,7 +6,7 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-public class DFSMazeSolverTest {
+public class DfsMazeSolverTest {
```

Comment:
Missed `classUnderTest` initialization. Better to copy/paste from the other repo.

Reply:
Also, references to this class in the `rst`s.

Reply:
refactor did blow on the deeeeeep files